docs/package.rst

```diff
@@ -0,0 +1,10 @@
+.. _package:
+
+Package
+=======
+
+pgtransaction
+--------------
+
+.. automodule:: pgtransaction.transaction
```

Comment:
I'm assuming you can't do automobile on `pgtransaction` because it will document the Django functions too?

You might be better off just doing

```
.. autofunction:: pgtransaction.atomic
```

so that it documents the `atomic` function under the proper module